index.js

```diff
@@ -13,26 +13,26 @@ function Account (options) {
     throw new Error('options.url is required')
   }
 
+  var cacheKey = options.cacheKey || '_session'
   var state = {
-    cacheKey: options.cacheKey || '_session',
+    cacheKey: cacheKey,
     url: options.url,
-    validate: options.validate || function () {}
+    validate: options.validate || function () {},
+    session: getSession({cacheKey: cacheKey})
   }
 
-  getSession(state)
```

Comment:
Ideally, we don’t pass `state` into util methods ever. `state` is only for public APIs, internal methods should rather have simple `options` and we pass only what the respective helper method needs. 

So `getSession` now only reads out the current state of localStorage, I set `state.session` now outside of it